paho/client.go

```diff
@@ -70,8 +70,9 @@ type (
 		Session          session.SessionManager
 		autoCloseSession bool
 
-		AuthHandler Auther
-		PingHandler Pinger
+		AuthHandler   Auther
+		PingHandler   Pinger
+		defaultPinger bool
```

Comment:
Oops. This was supposed to be used to determine whether the client's `SetDebugLogger()` should call `SetDebug()` on the pinger. So right now, the `Pinger`s logger will won't be set if the user is relying on the default one. This would be a very small fix, so could be tacked on to some other PR.